Trading/Mode/dca_trading_mode/dca_trading.py

```diff
@@ -740,17 +768,34 @@ async def single_exchange_process_optimize_initial_portfolio(
         )
 
     async def single_exchange_process_health_check(self, chained_orders: list, tickers: dict) -> list:
-        created_orders = []
         common_quote = trading_exchanges.get_common_traded_quote(self.exchange_manager)
+        if (
+            self.exchange_manager.is_backtesting
+            or common_quote is None
+            or not (self.use_take_profit_exit_orders or self.use_stop_loss)
+        ):
+            # skipped when:
+            # - backtesting
+            # - common_quote is unset
+            # - not using take profit or stop losses, health check should not be used
+            return []
```

Comment:
Shouldn't we override `should_trigger_health_check`?

Reply:
should_trigger_health_check renamed in https://github.com/Drakkar-Software/OctoBot-Trading/pull/1006